src/games/supported/BreakoutSuper.hpp

```diff
@@ -0,0 +1,87 @@
+/* *****************************************************************************
+ * The lines 68 and 76 are based on Xitari's code, from Google Inc.
```

Comment:
@pvchaumier , is this true? If you copied and pasted a different ROM file, this will be wrong.